examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<blockquote>We also use the function <code>on_termination</code></blockquote><p>We also use a function <code>on_termination </code>(...)</p><p> </p><p>Termination functions are optional but useful if you want to perform something after an activity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Why a?  Is the function that we define here. </p>

Reply:
<p>The thing I dislike is that it is confusing whether you mean the parameter <code>on_termination_function</code> of register_activity or the function <code>on_termination</code> because you gave it the same name . It would be better, e.g., if you call it <code>finish_volcano_detection</code> and then explain that you can execute something when an activity finishes by setting a <code>on_termination_function. </code></p>

Reply:
Yes, I removed it, I agree. My comment was on the grammar (a vs the)